### DISCUSSION TITLE
Welcome to ios-screens Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [Dezenix/mobile-screens#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Don't Spam the discussion box.
  * Use of harsh word and disrespect for other will lead to ban

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Daksh-jha** [comment]
Sir, I cannot find discord server link , for which I have asked in mail.

**Reply by ghost** [reply]
discord.io/dezenix

**Reply by Daksh-jha** [reply]
Thankyou Sir
